tests/backgroundTestServers.py

```diff
@@ -49,11 +50,35 @@ def runSubjectService(args, isStartedEvent):
     isStartedEvent.set()
 
 
+def runExampleService(args, isStartedEvent):
+    exampleServer = ExampleService(args)
+    exampleThread = threading.Thread(
+        name='exampleThread',
+        target=exampleServer.wsRemoteService.runForever
+    )
+    exampleThread.start()
+    while exampleServer.wsRemoteService.started is False:
+        time.sleep(.1)
+    isStartedEvent.set()
+
+
+defaultCfg = StructDict({'sessionId': "test",
+                         'subjectName': "test_sample",
+                         'subjectNum': 1,
+                         'subjectDay': 1,
+                         'sessionNum': 1})
+
+defaultProjectArgs = StructDict({'config': defaultCfg,
+                                 'mainScript': 'projects/sample/sample.py',
+                                 'port': 8921,
```

Comment:
This port seems like it's used frequently -- maybe factor it out to the `common.py` as a constant?